db_setup.sql

```diff
@@ -0,0 +1,26 @@
+USE apartments_test;
+
+DROP TABLE IF EXISTS Review;
+DROP TABLE IF EXISTS User;
+
+CREATE TABLE User (
+UserId VARCHAR(50) NOT NULL,
+FirstName VARCHAR(50) NOT NULL,
+LastName VARCHAR(50) NOT NULL,
+PRIMARY KEY (`UserId`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+CREATE TABLE Review (
+ReviewId VARCHAR(50) NOT NULL,
+Date DATE,
+Rating VARCHAR(50),
+Description VARCHAR(500),
+UserId varchar(50) NOT NULL,
```

Comment:
This is nitpicky, but it might be better if UserId is the second attribute in the list here since that's how we usually write the primary keys and foreign keys in